src/aiai_eval/task_configs.py

```diff
@@ -18,8 +18,8 @@ def get_all_dataset_tasks() -> Dict[str, DatasetTask]:
 NER = DatasetTask(
     name="ner",
     dataset_name="dane",
-    pretty_dataset_name="the truncated version of DaNE",
-    huggingface_id="ScandEval/dane-mini",
+    pretty_dataset_name="The Danish Dependency Treebank",
```

Comment:
DDT consists of 3+ datasets, with DaNE being the NER part, so we need to specify what we mean here.

```suggestion
    pretty_dataset_name="DaNE",
```